src/libPOW/pow.cpp

```diff
@@ -457,8 +457,10 @@ void POW::InitOpenCL()
                                OPENCL_GLOBAL_WORK_SIZE_MULTIPLIER, 0,
                                OPENCL_START_EPOCH, 0, 0, false, false))
     {
-        throw std::runtime_error(
+        std::string msg(
             "Failed to configure OpenCL GPU, please check hardware");
+        LOG_GENERAL(FATAL, msg);
+        throw std::runtime_error(msg);
```

Comment:
Once you have LOG FATAL, you don't need throw.

Reply:
If don't throw, later the software will crash, and from the terminal user need to think about why the software is crashed.

Reply:
The LOG  FATAL will throw the error message.

Reply:
Hi @ShengguangXiao , if it cannot be resolved now, please create an issue for it. Let's finish this PR by today.